app/lib/feed_manager.rb

```diff
@@ -141,6 +141,8 @@ def filter_from_home?(status, receiver_id)
     return false if receiver_id == status.account_id
     return true  if status.reply? && (status.in_reply_to_id.nil? || status.in_reply_to_account_id.nil?)
 
+    return true if keyword_filter?(status, Glitch::KeywordMute.matcher_for(receiver_id))
```

Comment:
on thinking about this... prrrrrrrrrrobably should filter from mentions as well?

Reply:
I don't think I've written an example specifically for mentions, but AFAICT status text is present identically in both replies and non-replies.  Adding an example to prove or refute this is a good idea, though.

Reply:
Oh wait, are you referring to the Notifications timeline?  In that case, yeah, I could add that.

Reply:
yes, that

lol i had this exact text box open in a tab somewhere to clarify it

my fuckin brains...

filter_from_mentions is the notifications column

Reply:
Mention filtering added in d03b48cea06ac87eecaf7eae96d175ab8ee621ca.